_sections/120-entities.md

```diff
@@ -3,11 +3,22 @@ title: Entities
 page: entities
 ---
 
-Version 2022.1.0
+Version 2023.1.0
+
+⚠️ In this specification, collections of Entities are referred to as Datasets. The term "Entity List" is generally recommended instead of "Dataset" in text that is intended for users rather than developers.
 
 ### Introduction
 
-This specification is a sub-specification of the [ODK XForms Specification](./). It describes a semantic layer that identifies the subject of a form ("entity") and its properties. Consumers that implement this specification can process form submissions to extract entity information based on the directives provided in the form definition.
+This specification is a sub-specification of the [ODK XForms Specification](./). It describes a semantic layer that identifies the subject of a form ("Entity") and its properties. Consumers that implement this specification can process form submissions to extract Entity information based on the directives provided in the form definition.
+
+### Versions
+
+| Version  | Changes |
+|----------|-------------------------------------------------------------------------------------------------------------------|
+| 2023.1.0 | Adds Entity updates from form submissions, still with Entities only created on the server                         |
```

Comment:
Should `created` be something more like `modified`? "Still with Entities only modified on the server (no offline Entity updates)"

Reply:
Ah yes, I see! I was thinking there's nothing to update locally if Entity creation is only on the server but I see it's a bit strange. I changed it to "only created or updated on the server"